README.md

```diff
@@ -10,6 +10,9 @@
   <a href="https://github.com/ttytm/wthrr-the-weathercrab" target="_blank">
     <img alt="last-commit" src="https://img.shields.io/github/last-commit/ttytm/wthrr-the-weathercrab?style=flat-square" />
   </a>
+  <a href="https://github.com/ttytm/wthrr-the-weathercrab" target="_blank">
+    <img alt="msrv" src="https://img.shields.io/badge/MSRV-1.72.0-brightgreen?style=flat-square" />
```

Comment:
```suggestion
    <img alt="msrv" src="https://img.shields.io/badge/MSRV-1.72.0-CE9244?style=flat-square" />
```

For the color, lets use something more rust-like. 
While already updating the shields, the same color can be used for the crates.io shield and the sorting can be updated.

> ![Screenshot_20231008_214228](https://github.com/ttytm/wthrr-the-weathercrab/assets/34311583/9ba692d0-43ba-4c21-bc7d-20054cea30be)

When showing minimum supported versions, I'm thinking that a lower version can resemble compatibility and a higher version resembles that it takes advantage of newer technologies. Is it good to strike a balance or is it just the higher the better for rust?

Reply:
Just for completeness regarding the colours:  the official Linguist colour for Rust is 0xDEA584.  Maybe this might be an option, as well, in order to have consistency with the repository statistics.

Reply:
Regarding the version, it is literally just a matter of style.  New versions deny more bad stylistic concepts than older ones *and* allow for access to more features which were formerly considered Nightly only but now have become stable.  Thus, I prefer for *my* projects to always have the very latest Rust version set.

But after all:  your repository, your rules.  It is also a sane decision to name an older, stable version of Rust such that users do not always have to `rustup update` their installations (or have to wait for their package registries deploying the latest Rust toolchain).

In my opinion, this is the most important question to decide about.

Reply:
> Just for completeness regarding the colours: the official Linguist colour for Rust is 0xDEA584. Maybe this might be an option, as well, in order to have consistency with the repository statistics.

Yes I know I'm deviating from the official colour. The manual adjustment was as the official colour code results in black text and it looked off to have a single badge with black text - I checked just changing the MSRV badge yesterday. But testing it again now for the two badges I like it better. The Rust logo itself is usually black too.

So for the colour, let's use official code. Thanks for the suggestion @kevinmatthes  👍

Reply:
> Regarding the version, it is literally just a matter of style. New versions deny more bad stylistic concepts than older ones _and_ allow for access to more features which were formerly considered Nightly only but now have become stable. Thus, I prefer for _my_ projects to always have the very latest Rust version set.
> 
> But after all: your repository, your rules. It is also a sane decision to name an older, stable version of Rust such that users do not always have to `rustup update` their installations (or have to wait for their package registries deploying the latest Rust toolchain).
> 
> In my opinion, this is the most important question to decide about.

Since whtrr is just a binary, I would base the rust version on the one that is made available by defaiult via the official packages of latest ubuntu lts. 1.66 atm: https://packages.ubuntu.com/search?keywords=rust&searchon=names&suite=jammy&section=all . If wthrr currently requires a higher version, let's use the first compatible that comes after the ubuntu lts version.

For libs and projects that other code bases would work with, I would also opt for the very latest rust version in the latest libs version.

Reply:
I just adjusted the badges as you requested.

Regarding the MSRV determination, I tried around with a tool named `cargo-msrv` and found some options to automate the MSRV update.  I am going to explain them in the following.

(I would like to suggest to rather start a new conversation thread within the PR for the MSRV updating logic, so I will mark this one as resolved, for now.)

Reply:
Awesome thank you @kevinmatthes. Gonna check it out today :+1: